jdaviz/app.py

```diff
@@ -487,11 +486,6 @@ def load_data(self, file_obj, parser_reference=None, **kwargs):
         parser_reference : str or `None`
             The actual data parser to use. It must already be registered
             to glue's data parser registry. This is mainly for internal use.
-
-        **kwargs : dict
-            Additional keywords to be passed into the parser defined by
-            ``parser_reference``.
-
```

Comment:
I think this used to exist on main and accidentally got deleted here when changing from kwargs to referencing the helper?

Reply:
True! Fixing.